packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -31,17 +33,25 @@ export async function searchCQDirectoriesAroundPatientAddresses({
   patient,
   radiusInMiles = DEFAULT_RADIUS_IN_MILES,
   mustHaveXcpdLink = false,
+  _searchCQDirectoriesByRadius = searchCQDirectoriesByRadius,
 }: {
   patient: Patient;
   radiusInMiles?: number;
   mustHaveXcpdLink?: boolean;
+  _searchCQDirectoriesByRadius?: typeof searchCQDirectoriesByRadius;
```

Comment:
I tried to test a module's function while mocking another function on the same module but couldn't get it to work. Tried multiple diff approaches (including the one below that I used successfully before on a slight diff situation):

```ts
jest.mock("../search-cq-directory", () => ({
  ...jest.requireActual("../search-cq-directory"),
  searchCQDirectoriesByRadius: jest.fn(async () => []),
}));
```